fxgl/src/main/kotlin/com/almasb/fxgl/app/FXGLApplication.kt

```diff
@@ -60,6 +59,12 @@ import java.nio.file.Paths
 import java.time.LocalDateTime
 import javax.imageio.ImageIO
 import kotlin.system.measureNanoTime
+import java.io.ByteArrayInputStream
```

Comment:
Are these two new imports necessary? Would be great to not have anything `java.awt` in imports.